print/server-middleware/routes/pdfGeneratorNativeChrome.js

```diff
@@ -12,14 +12,21 @@ const { performance } = require('perf_hooks')
 const { URL } = require('url')
 const { loadNuxt, build } = require('nuxt')
 const { Router } = require('express')
+const { memoryUsage } = require('process')
 
 const router = Router()
 
 let lastTime = null
 function measurePerformance(msg) {
   const now = performance.now()
+  const memory = memoryUsage()
+
   if (lastTime !== null) {
-    console.log(`(took ${Math.round(now - lastTime)} millisecons)`)
+    console.log(
+      `(took ${Math.round(now - lastTime)} millisecons / using ${
+        memory.heapUsed / 1000000
+      }MB)`
+    )
```

Comment:
How long does it currently take and how much memory is used?

Reply:
On my local machine (printing Harry-Potter-Lager with default print config):
- Time 6024ms
- Memory total > 1200MB
- Memory difference between start and max: approx. 60MB

Total memory is of course way too much. Need to figure out what exactly is causing this (Nuxt? Development environment? Puppeter?). 

I guess this could be reduced by using only `puppeter-core` and not the full fledged `puppeter` (will come in separate PR).

```
ecamp3-print   | Building Nuxt if necessary... development
ecamp3-print   | ℹ Sentry reporting is disabled ("disabled" option has been set)
ecamp3-print   | ℹ Preparing project for development
ecamp3-print   | ℹ Initial build may take a while
ecamp3-print   | ℹ Discovered Components: .nuxt/components/readme.md
ecamp3-print   | ✔ Builder initialized
ecamp3-print   | ↻ Updated .nuxt/App.js
ecamp3-print   | ✔ Nuxt files generated
ecamp3-print   | ℹ Compiling Server
ecamp3-print   | ℹ Compiling Client
ecamp3-print   | ℹ Compiling Server
ecamp3-print   | ℹ Compiling Server
ecamp3-print   | ℹ Compiling Client
ecamp3-print   | ℹ Compiling Client
ecamp3-print   | ℹ Compiling Server
ecamp3-print   | ℹ Compiling Client
ecamp3-print   | ✔ Server: Compiled successfully in 3.50s
ecamp3-print   | ✔ Client: Compiled successfully in 4.15s
ecamp3-print   | ℹ Waiting for file changes
ecamp3-print   | (took 34751 millisecons / using 1200.53544MB)
ecamp3-print   |
ecamp3-print   |
ecamp3-print   | Rendering page in Nuxt...
ecamp3-print   | http://caddy:3001/camps/05ce4b9836e9 - execution time: 119 ms
ecamp3-print   | http://caddy:3001/categories?camp=%2Fcamps%2F05ce4b9836e9 - execution time: 109 ms
ecamp3-print   | http://caddy:3001/activities?camp=%2Fcamps%2F05ce4b9836e9 - execution time: 207 ms
ecamp3-print   | http://caddy:3001/schedule_entries?period=%2Fperiods%2Fe8c03e4285cb - execution time: 88 ms
ecamp3-print   | http://caddy:3001 - execution time: 76 ms
ecamp3-print   | http://caddy:3001/content_types - execution time: 66 ms
ecamp3-print   | http://caddy:3001/content_nodes?contentType=%2Fcontent_types%2F318e064ea0c9&period=%2Fperiods%2Fe8c03e4285cb - execution time: 130 ms
ecamp3-print   | http://caddy:3001/content_nodes?period=%2Fperiods%2Fe8c03e4285cb - execution time: 277 ms
ecamp3-print   | http://caddy:3001/material_lists?camp=%2Fcamps%2F05ce4b9836e9 - execution time: 85 ms
ecamp3-print   | (took 2470 millisecons / using 1254.885112MB)
ecamp3-print   |
ecamp3-print   |
ecamp3-print   | Connecting to puppeteer...
ecamp3-print   | (took 225 millisecons / using 1256.425712MB)
ecamp3-print   |
ecamp3-print   |
ecamp3-print   | Puppeteer set HTML content & load resources...
ecamp3-print   | (took 1964 millisecons / using 1262.237016MB)
ecamp3-print   |
ecamp3-print   |
ecamp3-print   | Generate PDf...
ecamp3-print   | (took 1365 millisecons / using 1255.374656MB)
```

Reply:
In production it's currently around 350MB. Seems a major portion comes from running Nuxt in dev mode.
Hope to bring it further down by using `puppeteer-core`.